README.md

```diff
@@ -1,5 +1,7 @@
 # DotNetDockerTest
 
+![.NET Core](https://github.com/LordNibbler274/DotNetDockerTest/workflows/.NET%20Core/badge.svg)
```

Comment:
This should allow to point to the right upstream repo, filter down the runs, and make it a link:

```suggestion
[![.NET Core](https://github.com/G-Research/DotNetDockerTest/workflows/.NET%20Core/badge.svg?branch=master&event=push)](https://github.com/G-Research/DotNetDockerTest/actions/workflows/dotnetcore.yml?query=branch%3Amaster+event%3Apush)
```